app/src/main/java/com/xinto/opencord/ui/widget/Message.kt

```diff
@@ -43,11 +43,18 @@ fun WidgetChatMessage(
         Color.Unspecified
     }
 
+    val isMerged = avatar == null && author == null
```

Comment:
I'd rather have this be a component paramater rather than inferred from the parameters

Reply:
guhhh xinto said i should do it this way

Reply:
guh xintoe big dumdum